packages/flutter/lib/src/material/dropdown.dart

```diff
@@ -1512,11 +1540,8 @@ class DropdownButtonFormField<T> extends FormField<T> {
 
   /// The decoration to show around the dropdown button form field.
   ///
-  /// By default, draws a horizontal line under the dropdown button field but can be
-  /// configured to show an icon, label, hint text, and error text.
-  ///
-  /// Specify null to remove the decoration entirely (including the
-  /// extra padding introduced by the decoration to save space for the labels).
```

Comment:
Don't we need to carry on explaining this detail about the extra padding?

Reply:
Heh. Yeah, that "feature" was never true.  If you specified null, it would assert.

Good point though: I deleted it, meaning to keep the assert, but then I decided that  it was a necessary feature to get the `focusColor` correct in the decorator. I added an appropriate description back in.